modules/apps/site/site-cms-site-initializer/test/js/main_view/default_permission/DefaultPermissionForm.test.tsx

```diff
@@ -286,4 +286,49 @@ describe('DefaultPermissionForm', () => {
 
 		expect(screen.getByRole(`textbox`, {name: /search/i})).toBeDisabled();
 	});
+
+	it('Disable guest unsupported checkboxes', async () => {
+		const props = {
+			actions: [
+				{guestUnsupported: true, key: 'DELETE', label: 'View'},
+				{guestUnsupported: true, key: 'UPDATE', label: 'Update'},
+				{key: 'VIEW', label: 'View'},
+			],
+			roles: [
+				{key: 'Admin', name: 'Administrator', type: '1'},
+				{key: 'Guest', name: 'Guest', type: '2'},
+			],
+		};
+
+		renderComponent(props);
+
+		props.roles.forEach((role) => {
```

Comment:
just do the expect without loop

like expect screen.getbyid('row-checkbox-Admin_DELETE).ISdisabled

other action same role isDisables
other action oher role is enabled

are just 2 * 3 asserts